site/assets/scss/_variables_project.scss

```diff
@@ -23,7 +23,7 @@ $gray-800: #333 !default;
 $gray-900: #222 !default;
 $black: #000 !default;
 
-$primary: #f2771a !default;
+$primary: $mk-dark !default;
```

Comment:
I don't think this `primary` color is actually used anywhere, and it's orange. So I changed it to our theme color which is the k8s blue.